jobs/dd-agent/templates/config/datadog.yaml.erb

```diff
@@ -485,7 +485,13 @@ additional_endpoints:
 process_config:
 #   Note: the Process Agent expects this to be a string
```

Comment:
It says that, but it used to be `true` below? Are we sure it's the right value?

Reply:
Yes, the agent was parsing it as a string anyway